official/vision/beta/projects/yt8m/yt8m_agg_models.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2016 Google Inc. All Rights Reserved.

```

Comment:
Could you move this file into yt8m/modeling/ folder? So that the folder structure look alike to beta/ folder.